docs/root/operations/admin.rst

```diff
@@ -258,6 +258,12 @@ modify different aspects of the server:
    :ref:`Listener <envoy_v3_api_msg_config.listener.v3.Listener>` is used to determine whether a listener 
    is inbound or outbound.
 
+   .. http:post:: /drain_listeners?graceful
+
+   When draining listeners, enter a graceful drain period prior to closing listeners. 
+   This behaviour and duration is determined by server options for the drain manager, and 
```

Comment:
Please ref link to the relevant config and CLI options.

Reply:
I couldn't figure out how to ref link to the server options, not sure if the ServerInfo proto is what you mean, but I linked the CLI options.